docs/list_of_examples.jl

```diff
@@ -13,6 +13,7 @@ examples_files = [
 
 filter!(x -> !occursin("topo.jl", x), examples_files)                       # currently broken, TODO: Fix me!
 filter!(x -> !occursin("dry_rayleigh_benard.jl", x), examples_files)        # currently broken, TODO: Fix me!
+filter!(x -> !occursin("heldsuarez.jl", x), examples_files)        # currently broken, TODO: Fix me!
```

Comment:
Adding this line skips the held Suarez example. We need to remove this line and add a dictionary entry below in this file (assuming we want to include held Suarez in the docs)

Reply:
Roger that.